android/runtime/v8/src/native/JavaObject.cpp

```diff
@@ -172,8 +174,9 @@ void JavaObject::MakeJavaStrong()
 			if (stored == NULL) {
 				// Sanity check. Did we get into a state where it was weak on Java, got GC'd but the C++ proxy didn't get deleted yet?
 				LOGE(TAG, "!!! OH NO! We tried to move a weak Java object back to strong, but it's aleady been GC'd by JVM! We're in a bad state! Key: %d", refTableKey_);
+			} else {
```

Comment:
So this fix should definitely be applied. I assume if the store ref is NULL, trying to delete the local ref will be very bad.